paper.bib

```diff
@@ -59,7 +59,7 @@ @article{strobel_when_2009
 }
 
 @article{markham_project_2011,
-	title = {Project {Based} {Learning} {A} {Bridge} {Just} {Far} {Enough}},
```

Comment:
@QuLogic Any chance you know how I can control this in Zotero? The line above is what is exported when I chose `BibTex`. The only other sensible choice is `BibLatex`, but that creates `title = {Project Based Learning A Bridge Just Far Enough},`, which still has the capitals. Not sure what the presence of the additional `{}` around words is for either.